jenkins/dataproc-custom-images/create_dataproc_image.sh

```diff
@@ -36,7 +36,7 @@ gcloud config set dataproc/region us-central1
 
 python generate_custom_image.py \
     --image-name "$OUTPUT_IMAGE_NAME" \
-    --dataproc-version "2.1.11-debian11" \
+    --dataproc-version "2.2.5-debian12" \
```

Comment:
Looks like the latest is `2.2.21-debian12`, not sure if we want that?

https://cloud.google.com/dataproc/docs/concepts/versioning/dataproc-release-2.2